### ISSUE TITLE
Actions failing with "Cannot read property 'user' of undefined"

### ISSUE THREAD

**Issue by namelivia** [peaceiris/actions-label-commenter#276]
**Describe the bug**
Since #272 was merged my workflows that included the `@v1` version of the action started failing with the following error:
`##[error]Action failed with error "Cannot read property 'user' of undefined"`

**Your YAML file**
```yaml
# This workflow will automatically post comments on PRs that have certain labels
name: Label Commenter
# Trigger the workflow on labels added or removed for PRs
on:
  pull_request:
    types:
      - labeled
jobs:
  comment:
    if: contains(github.event.label.name, 'my-label-name')
    runs-on: ubuntu-18.04
    timeout-minutes: 2
    steps:
      - uses: actions/checkout@v2
        with:
          ref: master

      - name: Label Commenter
        uses: peaceiris/actions-label-commenter@v1
        with:
          github_token: ${{ secrets.GH_TOKEN }}
```

**Additional context**
The execution log shows:
```
Run peaceiris/actions-label-commenter@v1
[INFO] config_file: .github/label-commenter-config.yml
[INFO] labelName: my-label-name
[INFO] labelEvent: labeled
[INFO] issueNumber: 11002  
[INFO] commentBody: My comment content
[INFO] finalAction: undefined  
##[error]Action failed with error "Cannot read property 'user' of undefined"
```
So the issue seems to be defined as it gets the issue number. I believe the problem happens [here](https://github.com/peaceiris/actions-label-commenter/pull/272/commits/2e07d2a8fc2a53da454036002c373f32fd682b9b#diff-8cfead41d88ad47d44509a8ab0a109adR133).

**Comment by peaceiris** [comment]
Please follow the latest README, use `pull_request_target` and `${{ secrets.GITHUB_TOKEN }}`. Do not use your personal access token.

```diff
-  pull_request:
+  pull_request_target:
```

```diff
-          github_token: ${{ secrets.GH_TOKEN }}
+          github_token: ${{ secrets.GITHUB_TOKEN }}
```

**Comment by namelivia** [comment]
Hello! Firstly thanks for the fast answer, I've updated my workflow to this:
```
# This workflow will automatically post comments on PRs that have certain labels
name: Label Commenter
# Trigger the workflow on labels added or removed for PRs
on:
  pull_request_target:
    types:
      - labeled
jobs:
  comment:
    if: contains(github.event.label.name, 'my-label-name')
    runs-on: ubuntu-18.04
    timeout-minutes: 2
    steps:
      - uses: actions/checkout@v2
        with:
          ref: master

      - name: Label Commenter
        uses: peaceiris/actions-label-commenter@v1
        with:
          github_token: ${{ secrets.GITHUB_TOKEN }}
```
And I'm still getting the same error. I'm trying to get more information so I can see what's going on.

**Comment by peaceiris** [comment]
Fixed by #278

**Comment by namelivia** [comment]
Fixed for me! Thank you very much!
